content/get-started/index.md

```diff
@@ -17,7 +18,8 @@ Create a demo directory to try out webpack. [Install webpack](/get-started/insta
 mkdir webpack-demo && cd webpack-demo
 npm init -y
 npm install --save-dev webpack
```

Comment:
Ironically I've never tried this before, do you really need to reference the full bin path? @bebraw @pksjce